projects/organization-management/src/app/models/organization/organization.mapper.ts

```diff
@@ -0,0 +1,22 @@
+import { OrganizationDocument } from './organization.interface';
+import { Organization } from './organization.model';
+
+export class OrganizationMapper {
+  static fromData(payload: OrganizationDocument): Organization {
+    const data = payload.data;
+
+    if (!data) {
+      throw new Error(`'organizationDocument' is required`);
+    }
+
+    return {
+      id: data.id,
+      name: data.attributes ? data.attributes.name : undefined,
+      description: data.attributes ? data.attributes.description : undefined,
+      authenticationUrl: data.attributes ? data.attributes.authenticationUrl : undefined,
+      nodes: data.relationships.nodes.data.map(identifier => identifier.id),
```

Comment:
Shouldn't that refer to the node.mapper.ts? So you could work with a node-tree for the given organization?

Reply:
Request contains not enough information to build up an node tree.